CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## Unreleased
+
+### Added
+
+- Use `setup.$viewport-elements` to define which element should be considered equal to the viewport in size. these elements’ heights will match the viewport, including any variable sizing on mobile browsers.
```

Comment:
so this would only work if all of the ancestor elements also have these styles right? 

```
html
body
-- #random-container
-----#app-container
```

so if the dom looks something like that, and i only pass `#app-container` would this still work? if not, maybe we can clarify?

Reply:
the intention here is that a wrapper element from a framework (like `#root` from react) can be added to the list. If there are multiple wrappers for some reason, they’d each need to be added to the list — I’ll make sure that’s clear in the docs, thanks!

(btw it does work with a second, unstyled wrapper in safari, but not in chrome or firefox: https://codepen.io/darrencadwallader/pen/GRBJReZ)